examples/envelope_designer/sketch.js

```diff
@@ -0,0 +1,103 @@
+var attackTime = 0.001
```

Comment:
I wonder if we should lower the range of the slider, or think of some other interface (perhaps a number input), since a value of 0.1 vs 0.001 second makes an audible difference? On the other hand, that would make the code more complex...